### PR TITLE
https://issues.redhat.com/browse/ACM-8438 Removing unsupported scenario

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-8438

### CODE REVIEW

**Review by birsanv — APPROVED** [review]
lgtm thanks

File: business_continuity/backup_restore/backup_activation.adoc [hunk 1]
```diff
@@ -8,30 +8,9 @@ See _Scheduling and restoring backups_ to learn how you can use the operator.
...
-kind: MultiClusterHub
-metadata:
-  name: multiclusterhub
-  namespace: <namespace>
-spec:
-  disableHubSelfManagement: true
-----
-
-*Note:* If the self managing option is not disabled on the restore hub cluster before the activation data is moved to the restore hub cluster, the `local-cluster` klusterlet and the klusterlet in the managed cluster namespace conflict. As a result, the restore hub cluster is managed by itself by using the managed cluster and by using the restored managed cluster. If you detach the managed cluster as part of the detach operation, the managed cluster receives a deprovision request, which results in restore hub cluster removal.
+//is there a command that the user should run to shut the cluster down? Is there a button they should select from the console?
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_activation.adoc]: @birsanv is there a step that the user needs to complete to shut down the cluster?

Comment (dockerymick) [reply business_continuity/backup_restore/backup_activation.adoc]: Screenshot 2023-11-02 at 11 12 03 AM

Comment (swopebe) [reply business_continuity/backup_restore/backup_activation.adoc]: Let's remove comment.

Comment (dockerymick) [reply business_continuity/backup_restore/backup_activation.adoc]: Yup that's the plan, just wanted you to see the convo

Comment (swopebe) [reply business_continuity/backup_restore/backup_activation.adoc]: @dockerymick I don't understand why we need this file anymore. It seems this info can be part of another back up concept??

Comment (swopebe) [reply business_continuity/backup_restore/backup_activation.adoc]: There is not task here with these removals, or am missing something?

Comment (dockerymick) [reply business_continuity/backup_restore/backup_activation.adoc]: Good point, there is not much info on this file. There is only a paragraph. I recommend that we move the Restoring managed cluster activation data section to the [Restoring a backup](https://github.com/stolostron/rhacm-docs/blob/2.9_stage/business_continuity/backup_restore/backup_restore.adoc) file.

**Comment by dockerymick** [comment]
@birsanv @swopebe I made more updates. Will you both re-review when you have the opportunity

**Review by birsanv — APPROVED** [review]
lgtm

**Review by swopebe — APPROVED** [review]

**Comment by dockerymick** [comment]
Thanks for the reviews and approvals! Merging this PR

**Pull request merged**